### ISSUE TITLE
Dropping a table gives a warning about index creation

### ISSUE THREAD

**Issue by williamdclt** [sbdchd/squawk#152]
Very likely a variant/duplicate of https://github.com/sbdchd/squawk/issues/151.

```bash
$ echo 'DROP TABLE IF EXISTS some_table' | npx squawk-cli
stdin:1:0: warning: require-concurrent-index-creation

   1 | D

  note: Creating an index blocks writes.
  help: Create the index CONCURRENTLY.

find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
```

**Comment by sbdchd** [comment]
thanks for the report and sorry for the delay, should be fixed in version 0.7.3
